### DISCUSSION TITLE
elapsed time is not true

### DISCUSSION THREAD

**Discussion by sergey-boreysha** [MusicPlayerDaemon/MPD#1907] [Q&A]
hello, I need your help
I'm using your libmpd library in a home project and encountered a problem getting elapsed time:
elapsed_time = mpd_status_get_elapsed_song_time(obj);
the sound from the speakers does not coincide with the program's response in time
what could this be connected with?

**Comment by MaxKellermann** [comment]
libmpd is not supported by this project; it is a library only used by gmpc.

**Comment by sergey-boreysha** [comment]
Tell me, does gmpc allow you to get the elapsed time on request?
